src/main/include/RobotContainer.h

```diff
@@ -17,9 +19,13 @@
 
 class RobotContainer {
 public: // PROPERTIES
-    std::shared_ptr<t34::T34XboxController> DriveController;
-    std::shared_ptr<t34::SwerveDrive>  SwerveDrive;
+    std::shared_ptr<t34::T34XboxController> ctrl;
+    std::shared_ptr<t34::SwerveDrive>  swerve_drive;
 
+    t34::Shooter shooter;
+    t34::Climber climber;
+
+    double arm_angle_setpoint;
```

Comment:
Suggest either adding the units as a suffix or using the units library:

```suggestion
    double arm_angle_setpoint_degrees;
```
... or ...

```suggestion
    units::angle arm_angle_setpoint;
```